src/test/java/com/listywave/acceptance/collection/CollectionAcceptanceTest.java

```diff
@@ -1,5 +1,110 @@
 package com.listywave.acceptance.collection;
 
-// TODO: 정수씨 테스트 부탁해요
-public class CollectionAcceptanceTest {
+import static com.listywave.acceptance.collection.CollectionAcceptanceTestHelper.나의_콜렉션_조회_API_호출;
+import static com.listywave.acceptance.collection.CollectionAcceptanceTestHelper.콜렉트_또는_콜렉트취소_API_호출;
+import static com.listywave.acceptance.collection.CollectionAcceptanceTestHelper.콜렉트_취소하기;
+import static com.listywave.acceptance.common.CommonAcceptanceHelper.HTTP_상태_코드를_검증한다;
+import static com.listywave.acceptance.list.ListAcceptanceTestHelper.가장_좋아하는_견종_TOP3_생성_요청_데이터;
+import static com.listywave.acceptance.list.ListAcceptanceTestHelper.리스트_저장_API_호출;
+import static com.listywave.acceptance.list.ListAcceptanceTestHelper.회원용_리스트_상세_조회_API_호출;
+import static com.listywave.list.fixture.ListFixture.지정된_개수만큼_리스트를_생성한다;
+import static com.listywave.user.fixture.UserFixture.동호;
+import static com.listywave.user.fixture.UserFixture.정수;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.http.HttpStatus.FORBIDDEN;
+import static org.springframework.http.HttpStatus.NO_CONTENT;
+
+import com.listywave.acceptance.common.AcceptanceTest;
+import com.listywave.collection.application.dto.CollectionResponse;
+import com.listywave.collection.application.dto.CollectionResponse.CollectionListsResponse;
+import com.listywave.list.application.domain.list.ListEntity;
+import com.listywave.list.application.dto.response.ListCreateResponse;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+@DisplayName("콜렉션 관련 인수테스트")
+public class CollectionAcceptanceTest extends AcceptanceTest {
+
+    @Test
+    void 타인의_리스트를_성공적으로_콜렉트한다() {
+        // given
+        var 동호 = 회원을_저장한다(동호());
+        var 정수 = 회원을_저장한다(정수());
+        var 정수_엑세스_토큰 = 액세스_토큰을_발급한다(정수);
+        var 동호_엑세스_토큰 = 액세스_토큰을_발급한다(동호);
+        var 리스트_생성_요청_데이터 = 가장_좋아하는_견종_TOP3_생성_요청_데이터(List.of());
+        리스트_저장_API_호출(리스트_생성_요청_데이터, 정수_엑세스_토큰).as(ListCreateResponse.class);
+
+        // when
+        var 응답 = 콜렉트_또는_콜렉트취소_API_호출(동호_엑세스_토큰, 1L);
+        var 정수_리스트_콜렉트수 = 회원용_리스트_상세_조회_API_호출(정수_엑세스_토큰, 1L).collectCount();
+
+        // then
+        HTTP_상태_코드를_검증한다(응답, NO_CONTENT);
+        assertThat(정수_리스트_콜렉트수).isEqualTo(1);
+    }
+
+    @Test
+    void 나의_리스트는_콜렉트할_수_없다() {
+        // given
+        var 정수 = 회원을_저장한다(정수());
+        var 정수_엑세스_토큰 = 액세스_토큰을_발급한다(정수);
+        var 리스트_생성_요청_데이터 = 가장_좋아하는_견종_TOP3_생성_요청_데이터(List.of());
+        리스트_저장_API_호출(리스트_생성_요청_데이터, 정수_엑세스_토큰);
+
+        // when
+        var 응답 = 콜렉트_또는_콜렉트취소_API_호출(정수_엑세스_토큰, 1L);
+        var 정수_리스트_콜렉트수 = 회원용_리스트_상세_조회_API_호출(정수_엑세스_토큰, 1L).collectCount();
+
+        // then
+        HTTP_상태_코드를_검증한다(응답, FORBIDDEN);
+        assertThat(정수_리스트_콜렉트수).isEqualTo(0);
```

Comment:
```suggestion
        assertThat(정수_리스트_콜렉트수).isZero();
```
로도 가능해요 ㅎㅎ

둘의 차이는 제가 느끼기엔 취향 차이라서 참고만 해주시면 될 것 같아요!

Reply:
꿀팁 감사함돠 ㅎㅎ